libcontainer/cgroups/systemd/v1.go

```diff
@@ -24,12 +24,15 @@ type legacyManager struct {
 	dbus    *dbusConnManager
 }
 
-func NewLegacyManager(cg *configs.Cgroup, paths map[string]string) cgroups.Manager {
+func NewLegacyManager(cg *configs.Cgroup, paths map[string]string) (cgroups.Manager, error) {
+	if cg.Rootless {
+		return nil, errors.New("can't use rootless systemd cgroups manager on cgroup v1")
```

Comment:
I think we use `unable to` in many such cases. `can not` or `cannot` is fine, too.

Current usage in runc codebase is (excluding vendor, including comments):
* cannot: 98
* can not: 2
* can't: 61
* unable to: 93

Reply:
I usually pick "unable to" for either errors on "our" side and/or "non fatal" situations (for more "fatal" errors, I pick `failed to`).

In this case  it looks like a "fatal" error, but not because "we (runc)" made a whoopsie?

I'm fine with either; thanks for searching those; looks like some low-hanging fruit to make it more consistent for a better user experience.

Reply:
Changed `can't` to `cannot` in two cases (first commit in this PR).